types/angular/jqlite.d.ts

```diff
@@ -651,6 +651,7 @@ interface JQuery {
 
     // Undocumented
     length: number;
+    [index: number]: TElement;
```

Comment:
Nor am I! Is the best thing for me to do is close this PR for now and open a new one if and when I find another solution?